src/App.tsx

```diff
@@ -1,61 +1,115 @@
-import './App.scss';
+import React, { useState } from "react";
+import "./App.scss";
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from "./api/users";
+import todosFromServer from "./api/todos";
+
+import Users from './types/Users';
+import TodosWithUsers from "./types/TodosWithUsers";
+
+import { TodoList } from "./components/TodoList";
 
 export const App = () => {
+  const [users] = useState<Users[]>(usersFromServer);
+  const [todos, setTodos] = useState<TodosWithUsers[]>(todosFromServer.map(todo => ({
+    ...todo,
+    user: users.find(user => user.id === todo.userId) || null
+  })))
+  const [title, setTitle] = useState<string>("");
+  const [userId, setUserId] = useState<number | undefined>(undefined);
+  const [titleError, setTitleError] = useState<boolean>(false);
+  const [userError, setUserError] = useState<boolean>(false);
+
+  const handleSubmit = (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+
+    if (!title || !userId) {
+      return;
+    }
+
+    const selectedUser = users.find((user) => user.id === userId);
+    if (!selectedUser) {
+      return;
+    }
+
+    const newTodo: TodosWithUsers = {
+      id: todos.length + 1,
```

Comment:
1) should add a title with the max existing ID + 1

This error means that you need to write a function that will loop through your to-do list and choose from there max id number. And you have to put id as maxId + 1;